import_craft.py

```diff
@@ -114,10 +114,15 @@ def smooth_object_meshes(self, ob):
     def read_parts_models(self, prefabs_dict, colliders, allow_no_material_mesh):
         for part in prefabs_dict.values():
             unselect_all_objects()
-            result = import_mu.import_mu(self, bpy.context, part['mu'], False)
-            if not result == {'FINISHED'}:
-                print('Warning : Error while importing file {}'.format(part['mu']))
-                return
+            try:
+                result = import_mu.import_mu(self, bpy.context, part['mu'], False)
+                if not result == {'FINISHED'}:
```

Comment:
it looks like almost the same test as in `__init__.py`; just test once (probably not the responsibility here)